File: C++/CLRS/Bubble_Sort.cpp
#include <bits/stdc++.h>
using namespace std;

void bubble_sort(int a[],int n)
{
    int i,j;
    int swp = 1;
    for (i=0;i<n-1 && swp;i++)
    {
        for (j=n;j>i;j--)
        {
            if (a[j]<a[j-1])
                swap(a[j],a[j-1]);
        }
    }
}

int main()
{
    int arr[]={7,2,9,3,5};
    bubble_sort(arr,5);

    int i;
    for (i=0;i<5;i++)
    {
        cout << arr[i] << ' ';
    }
}

File: Python/CLRS/Bubble Sort.py
def Bubble_Sort(num: list[int]):
    swap = 1
    for i in range(len(num)-1):
        if(swap):
            swap = 0
            for j in range(len(num)-1,i,-1):
                if num[j] < num[j-1]:
                    num[j],num[j-1]=num[j-1],num[j]
                    swap = 1

    return num

print(Bubble_Sort([1,2,3,4,5]))